docs/integrations/terraform/introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Statsig Terraform Provider
+---
+
+The Statsig Terraform Provider allows you to configure your gates and experiments with Terraform. This is achieved by leveraging the Console API. If there is something you need to perform that isn't supported by the Terraform Provider, checkout the [Console API](https://docs.statsig.com/console-api/introduction).
+
+It is hosted on the Terraform registry at https://registry.terraform.io/providers/statsig-io/statsig
+
+## Supported Features
+
+We currently only support a few Statsig configuration via Terraform, but are working on adding more.
```

Comment:
We currently only support the following Statsig configurations:

We will be adding Segments and Dynamic Configs shortly

Or something like that